third_party/move/tools/move-package/tests/test_runner.rs

```diff
@@ -130,6 +142,27 @@ pub fn run_test(path: &Path) -> datatest_stable::Result<()> {
     Ok(())
 }
 
+pub fn run_test(path: &Path) -> datatest_stable::Result<()> {
+    package_hooks::register_package_hooks(Box::new(TestHooks()));
+    if path
+        .components()
+        .any(|component| component == Component::Normal(OsStr::new("deps_only")))
+    {
+        return Ok(());
+    }
+
+    let output_v1 = run_test_impl(path, false)?;
+    let update_baseline = read_env_update_baseline();
+    let res_v1 = check_or_update(path, output_v1.clone(), update_baseline, false);
+    if read_env_var(ENABLE_V2) == "1" {
+        let output_v2 = run_test_impl(path, true)?;
+        if output_v1 != output_v2 {
+            // TODO: compare the result between V1 and V2.
```

Comment:
Comparison of V1 and V2 will be added later as explained in the description of this PR